tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -421,23 +425,66 @@ describe('firefox schema import', () => {
           ],
         },
       ];
-      expect(inner.normalizeSchema(schemas, 'url_overrides.json')).toEqual({
-        id: 'url_overrides',
-        types: {},
-        definitions: {
-          WebExtensionManifest: {
-            properties: {
-              chrome_url_overrides: { type: 'object' },
+      expect(inner.normalizeSchema(schemas, 'url_overrides.json')).toEqual([
+        {
+          id: 'url_overrides',
+          types: {},
+          definitions: {
+            WebExtensionManifest: {
+              properties: {
+                chrome_url_overrides: { type: 'object' },
+              },
             },
           },
-        },
-        refs: {
-          'url_overrides#/definitions/WebExtensionManifest': {
-            namespace: 'manifest',
-            type: 'WebExtensionManifest',
+          refs: {
+            'url_overrides#/definitions/WebExtensionManifest': {
+              namespace: 'manifest',
+              type: 'WebExtensionManifest',
+            },
           },
         },
-      });
+      ]);
+    });
+
+    it('resolves the top level $import in API namespace definitions', () => {
+      const schemas = [
+        {
+          namespace: 'action',
+          functions: [{ name: 'setTitle', type: 'function', parameters: [] }],
+          events: [{ name: 'onClicked', type: 'function', parameters: [] }],
+          // min/max_manifest_version are expected to not be imported in the
+          // jsonschema definition importing this one.
```

Comment:
```suggestion
          // JSONSchema definition importing this one.
```

(as you wrote it many times above :p )